lib/src/endpoints/search.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2018 hayribakici. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+part of spotify;
+
+class Search extends EndpointPaging {
+  @override
+  String get _path => 'v1/search';
+
+  Search(SpotifyApiBase api) : super(api);
+
+  BundledPages get(String searchQuery, [Iterable<SearchType> types = null]) {
```

Comment:
```suggestion
  BundledPages get(String searchQuery, [Iterable<SearchType> types = SearchType.all]) {
```

Using this value as the default lets you remove [line 13](https://github.com/rinukkusu/spotify-dart/pull/9/files#diff-0c82323ee2900634141bc633d4498757R13).